cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -294,10 +294,7 @@ func KnownControllers() []string {
 	return ret.List()
 }
 
-var ControllersDisabledByDefault = sets.NewString(
-	"bootstrapsigner",
-	"tokencleaner",
-)
+var ControllersDisabledByDefault = sets.NewString()
```

Comment:
By default, these controllers doesn't do anything. They just sit there and wait for the user to create the `cluster-info` configmap. But then, when that configmap is there, the controllers will help manage it.

AFAIU, the longer-term plan here is to create a way to walk up to a cluster and say "hey, please tell me about yourself", etc. and that's what the cluster-info CM will be used for.

I do think these controllers should always be on by default. It's still possible to disable them with `--controllers`. I don't really like have this as a double-opt-in (set both `--controllers` and create the CM).

These were disabled by default because of the alpha state.